src/frontend/qt_sdl/OSD.cpp

```diff
@@ -438,6 +448,36 @@ void DrawGL(float w, float h)
     glEnable(GL_BLEND);
     glBlendFunc(GL_ONE, GL_ONE_MINUS_SRC_ALPHA);
 
+    for (auto lo = LuaScript::LuaOverlays.begin(); lo != LuaScript::LuaOverlays.end();)
+    {
+        OverlayCanvas& overlay = *lo;
+        if (overlay.flipped)
+        {
+            glDeleteTextures(1,&overlay.GLTexture);
+            overlay.flipped=false;
+            overlay.GLTextureLoaded=false;
+        }
+        if (!overlay.GLTextureLoaded)
+        {
+            glGenTextures(1,&overlay.GLTexture);
```

Comment:
this is not what I meant. You only need to create a texture once with glGenTextures and glTexImage2D and then you can repeatedly change the texture's content with glTexSubImage2D.